manuscript/03.results.md

```diff
@@ -4,15 +4,15 @@
+To illustrate the unique value of lineage measurements when analyzing heterogeneous drug responses, we plotted a series of lineages from our measurements of the breast cancer cell line AU565 treated with gemcitabine (Fig. @fig:lineage). Gemcitabine is a chemotherapy agent that disrupts DNA replication and results in extension of and apoptosis in S phase [@pmid:11504793]. Observing a few random lineages in control condition and gemcitabine-treated cells demonstrates that there is a striking difference in the growth dynamics of individual cells, not only comparing the lineages in the two conditions, but also within each condition. From the prospective of population level analysis, studying the total number of cells at any given time although useful, cannot tell us anything about the heterogeneity going on among the cells. Investigating the single-cell lineage data we realize that within a given population of cells there are lineages growing fast, as well as cells that outlived the experiment time and did not divide once. Moreover, looking at each cell and those cells related to it at the same time, single-cell analysis shows a high correlation between the measured observations, for instance the duration of G1 phase is very close between sister cells, especially in control condition (Fig. @fig:lineage a), and the duration of each cell's phase is reflected through its dauther cell. These variation gets more complicated as we treat cells with different compounds (Fig @fig:lineage b). As a result, population-level drug response measurements provides an average of dynamical behaviors that mask variations existing in a population of cells over time. Measurements that record the history of cells (e.g., CFSE staining, Luria-Delbruck experiment) can help to identify the various dynamics within cell populations but must make assumptions about the dynamics of heterogeneity [@DOI:10.1073/pnas.1715639115; @PMID:28607484]. Lineage measurements, by contrast, provide sufficiently rich measurements to quantify the specific structure of heterogeneous response. By observing individual cells, a variety of phenotypic measurements such as inter-mitotic times, cell cycle phase durations, motility, cell death and division, morphology, and protein and transcription factor levels can be characterized in parallel [@doi:10.1126/science.abc1944]. 
```

Comment:
Please put this change on a separate branch.

Reply:
done.